.github/workflows/test.yml

```diff
@@ -2,11 +2,11 @@ name: pybikes
 
 env:
   PYBIKES_CYCLOCITY: ${{ secrets.PYBIKES_CYCLOCITY }}
-  PYBIKES_ECOBICI_BA_CLIENT_ID: ${{ secrets.PYBIKES_ECOBICI_BA_CLIENT_ID }}
-  PYBIKES_ECOBICI_BA_CLIENT_SECRET: ${{ secrets.PYBIKES_ECOBICI_BA_CLIENT_SECRET }}
```

Comment:
Expected unrelated change or mistake?

Reply:
Expected unrelated change 👌  this class was replaced by an unauthenticated one at https://github.com/eskerda/pybikes/pull/617